### DISCUSSION TITLE
Restrict download destination

### DISCUSSION THREAD

**Discussion by tkraft69** [sct/overseerr#2084] [Help]
I wish to monitor inbound downloads before they are moved to their final folders. Currently Overseerr will offer the user to select from the full list of indexed folders in Sonarr/Radarr.

Is there a way to limit this selection to a single fixed option for movies and simuílarily for series?

**Comment by TheCatLady** [comment]
You can set a default root folder in your Overseerr *arr settings, and not grant your users the advanced request options permission.

**Comment by mariomanic** [comment]
I would like to be able to manage both restrictions independently. The current user profiles are:

- Users who can select both quality and the root folder.
- Users who cannot select anything, as defaults are enforced.
- Users who can select the quality but are restricted from modifying download folders.

Any solution?
